authors/apps/ratings/tests/test_ratings.py

```diff
@@ -0,0 +1,95 @@
+from rest_framework.reverse import reverse
+from rest_framework import status
+import logging
+
+# local imports
+from .test_base import TestBase
+logger = logging.getLogger(__file__)
+
+
+class TestRatings(TestBase):
+    ''' Ratings test cases '''
+
+    def test_post_rating(self):
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.rater_token)
+        response = self.client.post(
+            reverse(
+                'ratings',
+                kwargs={
+                    "slug": self.slug}),
+            data=self.rating,
+            format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_post_bad_rating(self):
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.rater_token)
+        response = self.client.post(
+            reverse(
+                'ratings',
+                kwargs={
+                    "slug": self.slug}),
+            data=self.bad_rating,
+            format='json')
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
This could be improved by testing that the `response message` is, as expected, descriptive of why this was a `Bad request`.